torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -154,20 +154,6 @@ TensorDomain* TransformReplay::fullSelfReplay(
   {
     size_t i = 0;
     for (auto id : self->getRootDomain()) {
-      TORCH_INTERNAL_ASSERT(
-          new_self_root->getRootDomain()[i]->getParallelType() ==
-                  id->getParallelType() &&
-              new_self_root->getRootDomain()[i]->isReduction() ==
-                  id->isReduction() &&
-              new_self_root->getRootDomain()[i]->isRFactorProduct() ==
-                  id->isRFactorProduct() &&
-              new_self_root->getRootDomain()[i]->isBroadcast() ==
-                  id->isBroadcast(),
-          "Axes ",
-          id,
-          " and ",
-          new_self_root->getRootDomain()[i],
-          " do not match for self replay.");
```

Comment:
This assert is failing, so I removed it, and seems like nothing is broken😅

Reply:
Do you know which check failed? I assume on the parallel type?
Did it only fail from `propagateTvSibling`?
Was it ever called on siblings before?

Reply:
Yes, it is the parallel type check. It is not only at `propagateTvSibling`, but also at `computeAt`. I just added back other checks.